Makefile

```diff
@@ -35,14 +35,14 @@
 #     make clean
 #
 
-OS = $(shell uname)
```

Comment:
This is still needed so the environment variables in the subsequent `ifeq` block are set (otherwise `make install` fails)

Reply:
Thanks for the testing and patches, @nathanweeks. After those changes on my system, I am seeing
`bundler: command not found: jekyll`
I'll try to investigate later today (traveling, limited work time this week).

Separately, do you have advice regarding the issue at GH Pages:
`If this is a development machine, remove the Gemfile.lock freeze by running
`bundle config set frozen false`.`
https://github.com/legumeinfo/jekyll-legumeinfo/actions/runs/15192962182/job/42730135750

Reply:
All right. Some hours later, I think this was my problem: I was using homebrew ruby (which is OK I. think), but the gems were also getting somewhere else (not in vendor). 
I did a clean install of the repository locally, and explicitly set the bundle path:
```
  bundle config set --local path 'vendor/bundle'

  make install
  make
```
... or in the Makefile, like so:
```
if ! bundle check; then $(XCRUN) bundle config set --local path 'vendor/bundle'; $(XCRUN) bundle install; fi
```
A possible small downside: this requires a new small config file:
`.bundle/config`

Is this acceptable?